### PR TITLE
Remove ImportContact screen

### LINKED ISSUE #1052: Remove Contact matchmaking screen
### Current behavior
There is a screen to import contacts that Google is complaining about.

### Desired behavior
There is no screen to sync contacts.

### Steps to Reproduce
Verify your phone number, see that you jump to the success screen directly.

### Versions
All

### PR DESCRIPTION
**Pull request by gnardini**
### Description

We are removing the ImportContactsScreen because of a violation of Google's policies

### Other changes

N/A

### Tested

Manually

### How others should test

Verify your phone number, see that you skip the import contacts screen.

### Related issues

- Fixes #1052

### Backwards compatibility

N/A

### CODE REVIEW

File: packages/mobile/locales/en-US/nuxNamePin1.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
...
+    "privacy": "By joining this network, you give us permission to collect anonymous information about your use of the app. Additionally, if you confirm your phone number, a hashed copy of it will be stored on the Celo Network. If you grant Valora access to your contact list, Valora will import contacts' name, phone number and profile picture to allow users to connect through the Valora app. To learn how we collect and use this information please review our <0>Privacy Policy</0>.",
```
Comment (silasbw) [inline packages/mobile/locales/en-US/nuxNamePin1.json:13]: i'm still confused about the grammar here.  specifically `Valora will import contacts' name`.  we're missing an article, or the plurality is wrong, or I'm just confused.  the previous version seemed clear to me:  `Valora will import each contact's name`

Comment (tarikbellamine) [reply packages/mobile/locales/en-US/nuxNamePin1.json]: Done

File: packages/mobile/locales/en-US/global.json [hunk 2]
```diff
@@ -201,5 +201,6 @@
   "cashOutLimitExceeded": "You cannot cash out more than your balance of {{balance}} {{currency}}",
   "appDescription": "A mobile payments wallet that works worldwide",
   "earn": "Earn",
-  "free": "Free"
+  "free": "Free",
+  "nowNow": "Not Now"
```
Comment (silasbw) [inline packages/mobile/locales/en-US/global.json:205]: naming typo (nowNow vs notNow)

Comment (tarikbellamine) [reply packages/mobile/locales/en-US/global.json:205]: Fixed thanks

**Review by silasbw — APPROVED** [review]

**Pull request merged**